### PR TITLE
Allow administrators to deactivate dossiers.

### PR DESCRIPTION
**Pull request by njohner**
Administrators cannot deactivate dossiers although they can for example activate a deactivated dossier. This is confusing for the clients and leads to support issues. I don't think that this is an intended feature so I would simply give that permission to administrators.

For https://extranet.4teamwork.ch/support/gemeinde-muri-guemligen/tracker/9

## Checkliste

_Zutreffendes soll angehakt stehengelassen werden._
- [x] Profil angepasst? Sind UpgradeSteps vorhanden/nötig?:
- [x] Sind UpgradeSteps `deferrable`, oder können gewisse Schritte des Upgrades konditional ausgeführt werden? Not deferrable
- [x] Changelog-Eintrag vorhanden/nötig?

### CODE REVIEW

File: opengever/core/upgrades/20200217115450_allow_administrators_to_deactivate_dossiers/upgrade.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+from ftw.upgrade import UpgradeStep
+
+
+class AllowAdministratorsToDeactivateDossiers(UpgradeStep):
+    """Allow administrators to deactivate dossiers.
+    """
+
+    def __call__(self):
+        self.install_upgrade_profile()
```
Comment (deiferni) [inline opengever/core/upgrades/20200217115450_allow_administrators_to_deactivate_dossiers/upgrade.py]: i think you also need to `update_workflow_security` here

Comment (njohner) [reply opengever/core/upgrades/20200217115450_allow_administrators_to_deactivate_dossiers/upgrade.py]: Ah sorry, you're right

Comment (njohner) [reply opengever/core/upgrades/20200217115450_allow_administrators_to_deactivate_dossiers/upgrade.py]: Actually I don't think we need to. This is a guard expression, not a permission stored on the object.

Comment (deiferni) [reply opengever/core/upgrades/20200217115450_allow_administrators_to_deactivate_dossiers/upgrade.py]: oh yes, you're right.

**Comment by njohner** [comment]
Same here, client deemed this as not urgent, so no backport needed

**Pull request merged**